to-do-list/src/App.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import Item from "./components/Item";
+
```

Comment:
You have to clear input-field after each time you add a todo

https://user-images.githubusercontent.com/34886027/210862454-66d2410a-5ba6-43ec-ba82-15345a7c9c60.mov